src/core/Format.java

```diff
@@ -0,0 +1,55 @@
+package core;
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.File;
+
+/**
+ * This class export a image with other format.
+ *
+ * @autor Bruno Vasquez
+ */
+public class Format {
```

Comment:
Probably you can name your class better, like ImageConverter, ImageFormat, ImageExport, etc